.zuul.yaml

```diff
@@ -37,3 +38,24 @@
       A zuul job to validate the watcher operator and its service deployment.
     vars:
       run_tempest: false
+
+- job:
+    name: watcher-operator-kuttl
+    parent: cifmw-multinode-kuttl-operator-target
```

Comment:
Content provider dependency is added at parent job.

Reply:
Here is the logs of Kuttl tests runs: https://logserver.rdoproject.org/14/14/5340526d457f271736200032a8125ed82f36c171/github-check/watcher-operator-kuttl/accd4a0/controller/controller/kuttl-test-run.out